my_list.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+end
+
+def each(&block)
+  @list.each(&block)
```

Comment:
- [ ] You have done good progress, unfortunately, you are sill using the `each` method given by ruby, please create your own `each` method without using it, try using `while` to iterate trough the array of arguments 😄

Reply:
could you provide my link to an example or the documentation, please?